packages/pharos/src/components/image-card/PharosImageCard.react.stories.tsx

```diff
@@ -519,8 +523,8 @@ export const SelectableCollection = {
           title="Selected Disabled"
           link="#"
           variant="selectable-collection"
-          disabled="true"
-          selected="true"
+          disabled
+          _isSelected
```

Comment:
This looks odd to me, in that we are setting what looks to be a private attribute via the public interface? Non-blocking just calling it out